test/problems/multipass/problem.yaml

```diff
@@ -1,8 +1,11 @@
 name: multi-pass
-author: Michael Zündorf
-source:
-source_url:
 uuid: 71076b69-e9c2-4227-ba54-ec3d4e277c78
 license: unknown
 rights_owner:
-validation: custom multi-pass
+problem_format_version: 2023-07-draft
+type:
+  - pass-fail
+  - multi-pass
```

Comment:
Same here, this could be `type: multi-pass`, right?

Reply:
yea I will change this to only add `pass-fail` if nothing else is in type